src/utils/UserUtils.ts

```diff
@@ -4,20 +4,28 @@ export const getUserType = (user: User) => {
   if (user.type === 'pupil') {
     return `Schüler*in`;
   }
+  return `Student*in`;
+};
+
+export const getUserTags = (user: User) => {
+  const studentTags = {};
+  if (user.type === 'pupil') {
+    return { pupil: 'Schüler*in' };
+  }
 
   if (user.isTutor) {
-    return `Student*in`;
+    studentTags["isTutor"] = 'Tutor';
   }
 
   if (user.isInstructor) {
-    return 'Tutor*in';
+    studentTags["isInstructor"] =
+      user.university === null ? 'Kursleiter' : 'Praktikant';
```

Comment:
I don't think that only having university info suffices to be an intern (aka "Praktikant\*in"). What do you think @lsc0498? I'm not sure the current backend API is providing that information. And also due to the triple equal, we need to notice that `user.university` being undefined also results in being a "Praktikant\*in".

Reply:
Actually I assumed that this would be a sufficient condition. Currently we have no other possibility to distinguish between `Praktikant*in` and `Kursleiter*in` given the existing backend api.
Concerning the triple equal, you're right we should use `user.university ?  'Praktikant' : 'Kursleiter'` instead.

Reply:
Ok. So if a university student registers for both being 1-on-1 tutor and course instructor (and not selecting the intern-flag and thus not providing the required information for interns), following those conditions here, she will be labelled as intern. Obviously this is wrong. 

But long story short: it looks like the backend has to provide the intern information - otherwise the frontend has no way to get this information here. So leaving it as proposed, there will be many people whose status will be displayed inappropriately.

Reply:
@geroembser and  @lsc0498 you're right about the university, we shouls have made more general, but we tested for the value none, bec. we found out that it was always set to none when it was empty, and never was it set to undefined, but i guess the more general the better

Reply:
@geroembser so would you that we revert back to the previous way, we used to handle things (not showing prakitikant), until we provide new attributes on the backend side

Reply:
If there is some doubt about whether the university field is sufficient to determine whether a Student is `Praktikant*in` or not I would suggest to write `Kursleiter*in` in either case for now. At least it wouldn't be totally wrong I guess 😉😅